src/components/DropDownMenu.js

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
+import Checked from '../images/iconfinder_done-01_186405.svg'
+import CheckedBlack from '../images/iconfinder_icon-checkmark_black.svg'
+export const DropDownMenu = ({ isOpen, setFormat, format, formatOptions, settings: { dark } }) => {
+  const imgToShow = dark ? Checked : CheckedBlack
```

Comment:
Since this is a simple statement used only once, you can inline it instead of creating a single-use variable.